minisom.py

```diff
@@ -546,6 +547,16 @@ def test_activation_reponse(self):
     def test_activate(self):
         assert self.som.activate(5.0).argmin() == 13.0  # unravel(13) = (2,3)
 
+    def test_distance_from_weights(self):
+        self.som._weights = arange(2 * 3 * 4).reshape(2, 3, 4)
+        data = arange(2 * 4).reshape(2, 4)
+        assert (self.som._distance_from_weights(data) == array([[0., 8., 16., 24., 32., 40.], [8., 0., 8., 16., 24., 32.]])).all()
+
+    def test_distance_from_weights_big_data(self):
+        self.som._weights = random.rand(50, 30, 300)
+        data = random.rand(10000, 300)
```

Comment:
Tests need to be minimalistic and repeatable. It's more valuable to test edge cases rather than the cases in which we're sure about the behaviour.

In the method `setUp` there is a some already defined. Please use that.

Reply:
The purpose of this test is to see the performance under big data. Do you want me to just remove this test?